IceCubesApp/App/AppRouter.swift

```diff
@@ -32,6 +32,7 @@ extension View {
         ConversationDetailView(conversation: conversation)
       case let .hashTag(tag, accountId):
         TimelineView(timeline: .constant(.hashtag(tag: tag, accountId: accountId)), scrollToTopSignal: .constant(0), canFilterTimeline: false)
+      // TODO: anything to do in the app router????
```

Comment:
You added your route so it seems all good. This todo can be deleted?

Reply:
@Dimillian yes I added a route for the "add screen" but I was not sure if a route was needed for the "tag group timeline" itself.